### ISSUE TITLE
Spotify Error at start of nuvola player

### ISSUE THREAD

**Issue by Chendalein** [tiliado/nuvolaplayer#741]
At first start I filled in my sotify user and password and afterwards I got an error message with following wording:
Irgendwas ist schiefgelaufen.
Versuche, die Seite erneut zu laden.

After restarting of Spotify nuvola player I get the same message without any more user interaction. 
I only can close the player but after restart I see the same issue.

I use Nuvola Player Version 4.22.0 (0-gcff8d27)
And Spotify script version 3.5.2 (2-gb8cdb39)
on Ubuntu 18.04 x86_64

**Comment by fenryxo** [comment]
Hello @Chendalein. Thanks for reporting your issue. I need more information to investigate it further. Could you [visit the Bug reporting guidelined](https://github.com/tiliado/nuvolaruntime/wiki/Bug-Reporting-Guidelines#basic-information) and answer the questions below?

* Is it the first time you launched Spotify in Nuvola?
* Does Spotify load correctly in a web browser?
* Could you obtain a debugging log and attach it here as a file?

**Comment by Chendalein** [comment]
Hi @fenryxo ,

thank you for answering.

> * Is it the first time you launched Spotify in Nuvola?
yes it is. I installed nuvola and the scripts for deezer, amazon music and spotify just yesterday. Deezer and amazon music work fine, but spotify not.

> * Does Spotify load correctly in a web browser?
yes it does. I tried it a minute ago. Site loads. Login works and music is playing.
> * Could you obtain a debugging log and attach it here as a file?
Not necessary anymore.
After I logged in to the website because of your question I started nuvola spotify parallel and it worked and showed, that I play in my Browser. Very strange for me but it seem to work now.
